functest_requirements.txt

```diff
@@ -2,3 +2,4 @@ pulp-smash @ git+https://github.com/pulp/pulp-smash.git
 orionutils
 pytest
 dynaconf
+ansible @ git+https://github.com/ansible/ansible.git@devel
```

Comment:
The installation signature verification is currently only available on the devel branch.

Reply:
To be upfront: This is only a test requirement. So we are fine with relying on some unreleased piece of code.